irohad/consensus/yac/storage/impl/yac_proposal_storage.cpp

```diff
@@ -114,36 +81,27 @@ namespace iroha {
         return true;
       };
 
-      bool YacProposalStorage::hasRejectProof() {
+      nonstd::optional<Answer> YacProposalStorage::findRejectProof() {
         // todo implement
-        return false;
+        return nonstd::nullopt;
       };
 
       uint64_t YacProposalStorage::findStore(ProposalHash proposal_hash,
```

Comment:
`uint64_t` can be replaced with `auto`, and method can return iterator.
In this case, instead of
```C++
auto index = findStore(msg.hash.proposal_hash, msg.hash.block_hash);
auto block_state = block_storages_.at(index).insert(msg);
```
we can use
```C++
auto it = findStore(msg.hash.proposal_hash, msg.hash.block_hash);
auto block_state = it->insert(msg);
```